egov-searcher/src/main/java/org/egov/SearchApplicationRunnerImpl.java

```diff
@@ -45,27 +51,84 @@
     @Override
     public void run(final ApplicationArguments arg0) throws Exception {
     	try {
-				logger.info("Reading yaml files......");			
-			    readFiles();			
+				logger.info("Reading yaml files......");
+			    readFiles();
 			}catch(Exception e){
 				logger.error("Exception while loading yaml files: ",e);
 			}
     }
-    
+
+	//file types to be resolved have to be passed as comma separated types.
+	public List<String> resolveAllConfigFolders(List<String> listOfFiles, Boolean resolveFolderCheck, String fileTypesToResolve){
+	List<String> ymlUrlList = new ArrayList<String>();
```

Comment:
Don't name this `yml` extension could be anything. Remove `resolveFolderCheck` also